packages/zowe-explorer/package.nls.json

```diff
@@ -67,6 +67,7 @@
   "Zowe-Default-Datasets-PS": "Default values of Sequential data set creation",
   "Zowe-Default-Datasets-Favorites": "Toggle if favorite files persist locally",
   "Zowe-Temp-Folder-Loacation": "Path to temporary folder location",
+  "zowe.hideTempFolder": "Hide the Zowe temporary folder from the workspace",
```

Comment:
@rudyflores @lauren-li do you have an updates list of settings names? Would it make sense to use the "files" group? I.e. `zowe.files.hideTempFolder` and `zowe.file.tempFolderPath` ?

Reply:
@crawr @phaumer The format being proposed for @JillieBeanSim's PR #1347 (Support multiple VS Code windows) is `zowe.files.temporaryDownloadsFolderCleanup`. I agree with the proposal in that PR, and I also think it makes sense to use the `files` group here in this PR. Below are the settings we could put in the `files` group thus far via @rudyflores' work:

- `zowe.files.temporaryDownloadsFolderCleanup`
- `zowe.files.tempFolderPath`(or `zowe.files.temporaryDownloadsFolderPath` as proposed in [this PR 1347 comment](https://github.com/zowe/vscode-extension-for-zowe/pull/1347#discussion_r660206692))
- `zowe.files.hideTempFolder`

We probably want to standardize on if we want to refer the temp folder as `tempFolder` or `temporaryDownloadsFolder`.

Reply:
I think to be consistent, we can go with `zowe.files.hideTemporaryDownloadsFolder` for this PR